l10n_ec_ote/models/ote.py

```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Canton(models.Model):
```

Comment:
This model should be replaced by res.city and depend on the module base_parish that is proposed on the partner-contact repository.